### PR TITLE
newbranch35

### PR DESCRIPTION
**Pull request by gudmdharalds**
Testing

### CODE REVIEW

File: bla-7.php [hunk 1]
```diff
@@ -1,3 +1,4 @@
 <?php
 
 echo "testing " . time() . PHP_EOL;
+echo "testing123 " . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-7.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time'.

File: bla-7.php [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+
+echo "testing321" . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-7.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time'.

File: bla-7.php [hunk 3]
```diff
@@ -1,3 +1,6 @@
 <?php
 
 echo "testing " . time() . PHP_EOL;
+echo "testing123 " . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-7.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time'.

File: bla-7.php (hunk 2 above)
Comment (gudmdharalds-a8c) [inline bla-7.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time'.

File: bla-7.php (hunk 3 above)
Comment (gudmdharalds-a8c) [inline bla-7.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time'.

File: bla-7.php [hunk 4]
```diff
@@ -1,3 +1,2 @@
 <?php
```
Comment (gudmdharalds-a8c) [inline bla-7.php]: :information_source: **Info**: File is approved in hashes-to-hashes database

File: bla-8.php [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<?php
+
+echo "testing" . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-8.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time'.

File: bla-7.php (hunk 4 above)
Comment (gudmdharalds-a8c) [inline bla-7.php]: :information_source: **Info**: File is approved in hashes-to-hashes database

File: bla-8.php [hunk 6]
```diff
@@ -0,0 +1,4 @@
+<?php
+// Silence is golden.
+
+echo "time:" . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-8.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time'.

File: bla-7.php (hunk 4 above)
Comment (gudmdharalds-a8c) [inline bla-7.php]: :information_source: **Info**: File is approved in hashes-to-hashes database

File: bla-8.php [hunk 7]
```diff
@@ -0,0 +1,4 @@
+<?php
+// Silence is golden.
+
+echo "testing:" . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-8.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time'.

File: bla-7.php (hunk 4 above)
Comment (gudmdharalds-a8c) [inline bla-7.php]: :information_source: **Info**: File is approved in hashes-to-hashes database

File: bla-8.php [hunk 8]
```diff
@@ -0,0 +1,3 @@
+<?php
+// Silence is golden.
+echo "test:" . time() . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline bla-8.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time'.

File: bla-8.php [hunk 9]
```diff
@@ -0,0 +1,6 @@
...
+
+echo "test:" . time() . PHP_EOL;
```
Comment (gudmdharalds) [inline bla-8.php]: Test

**Pull request closed without merging**